pygments/lexers/rust.py

```diff
@@ -66,7 +66,7 @@ class RustLexer(RegexLexer):
     tokens = {
         'root': [
             # rust allows a file to start with a shebang, but if the first line
-            # starts with #![ then it's not a shebang but a crate attribute.
+            # starts with #![ then it's not a shebang but a create attribute.
```

Comment:
This one is wrong, "crate" is the correct term here. (It's also a valid English word - how was this fix generated?)

Reply:
Done. Codespell and aspell.